src/pages/help-centre/_utility.js

```diff
@@ -14,3 +14,8 @@ export const splitArticles = (array, length) =>
         result[Math.floor(index / length)].push(item)
         return result
     }, [])
+
+export const euArticles = (array) => {
+    const second_array = [...array[1], array[2][0]]
+    return [array[0], second_array, [array[2][1]]]
+}
```

Comment:
Why are you manipulating the articles here?  This is a hardcoded way to manipulate the array and will be a problem if we were to change the articles array in the future

Reply:
@yashim-deriv this will only affect articles displayed in eu region at the moment, and in the future can be adjusted or removed

Reply:
Which articles that you want to hide for EU?

Reply:
is it under the `Deriv X` category? If it is, it is already covered by the condition in the map function. If it is not under `Deriv X` category the best way is to add a flag under that article, something like `hide_eu`. Doing it this way is very unreadable for those who will be working in the future.